packages/react-components/components/NumberKeypad.tsx

```diff
@@ -0,0 +1,86 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import Backspace from '@celo/react-components/icons/Backspace'
+import { fontStyles } from '@celo/react-components/styles/fonts'
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+interface Props {
+  showDecimal: boolean
+  onDigitPress: (digit: number) => void
+  onBackspacePress: () => void
+  onDecimalPress?: () => void
+}
+
+function DigitButton({
+  digit,
+  onDigitPress,
+}: {
+  digit: number
+  onDigitPress: (digit: number) => void
+}) {
+  const onPress = () => onDigitPress(digit)
+  return (
+    <Touchable borderless={true} onPress={onPress}>
+      <Text style={style.digit}>{digit}</Text>
+    </Touchable>
+  )
+}
+
+export default function NumberKeypad(props: Props) {
```

Comment:
I usually don't like custom keyboards as they break my expectations (can't use integration with password managers, or swipe style input).
Sure we really want our own? I understand it's for security reasons, but if the system keyboard is compromised I'm not entirely convinced a custom keypad will actually prevent much.

Reply:
Also, do custom keyboards have any trouble on low end phones?

Reply:
A lot of apps use custom keyboards for digits because the android numeric keyboard type varies so much across different phone vendors. And many look terrible.

@annakaz not sure why this component would cause problems on low end phones. If anything the experience is better because the component doesn't resize/redraw as the keyboard pops in and out